doc/examples/anim06/anim06.sh

```diff
@@ -14,15 +14,15 @@
 rate=24			# Frames per seconds
 frames=$(gmt math -Q 60 $rate MUL =)
 # 0. Initial parameters
-cat << EOF > init.sh
+cat << 'EOF' > init.sh
```

Comment:
We probably should not change this one, because we want to expand the variables `$rate` and `$frames`.

Reply:
OK, done